cmd/virt-api/virt-api.go

```diff
@@ -127,6 +127,14 @@ func (app *virtAPIApp) Run() {
 		ApiPath:         "/swaggerapi",
 		SwaggerPath:     "/swagger-ui/",
 		SwaggerFilePath: app.SwaggerUI,
+		Info: swagger.Info{
+			Title:       "KubeVirt API, ",
+			Description: "This is KubeVirt API an add-on for Kubernetes.",
+			//			TermsOfServiceUrl: "???",  // I don't know what to put there
```

Comment:
If you e.g. host a service, and you need to distribute usage-terms, this is the  place where you can specify a URL. I don't think it is important for us.

Reply:
OK U will remove it then